uplink/ecclient/client.go

```diff
@@ -122,9 +120,9 @@ func (ec *ecClient) Put(ctx context.Context, limits []*pb.AddressedOrderLimit, p
 
 		if info.err != nil {
 			if !errs2.IsCanceled(info.err) {
-				failures++
+				atomic.AddInt32(&failureCount, 1)
```

Comment:
Why do we need an atomic operation here?

Reply:
i don't think we need one here. this is on the stack, there are no concurrent accesses

Reply:
That's what I thought but I prefer to ask because I may miss something.

Reply:
i got carried away :p